spec/import_js/importer_spec.rb

```diff
@@ -1331,8 +1332,8 @@ def self.current_selection=(index)
             it 'adds the destructuring on a new line' do
               expect(subject).to eq(<<-EOS.strip)
 var _ = require('underscore');
-var foo = require('foo');
 var { memoize } = require('underscore');
+var foo = require('foo');
```

Comment:
A nice side-effect of this is now we are sorting based on `ImportStatement`s instead of the strings that they generate, which allows destructured requires to be next to the default requires.